frontend/apps/reader/readerui.lua

```diff
@@ -329,6 +330,7 @@ end
 
 function ReaderUI:showReader(file)
     DEBUG("show reader ui")
+    ReadHistory:addItem(file)
```

Comment:
Yeah, I am more interested in serialization cost and memory consumption for large tables. The current legacy history implementation is unfortunately unbouanded ;) Hopefully, we can get rid of it soon.